sources/packages/api/src/route-controllers/student/models/student.dto.ts

```diff
@@ -79,3 +79,46 @@ export interface StudentDetailDTO {
   pdStatus: StudentPDStatus;
   hasRestriction: boolean;
 }
+
+/**
+ * Enumeration for Document purpose these are
+ * the dropdown options from the form student
+ * uploader, for the dropdown down field
+ * `Document purpose`.
+ *
+ */
+export enum DocumentPurpose {
+  Application = "application",
+  Verification = "verification",
+  Appeal = "appeal",
+  Identity = "identity",
+}
+
+/**
+ *  FormIO FileObject
+ */
+export interface FormIOFileObject {
```

Comment:
Can you please elaborate on why do we need this interface?

Reply:
it is for file objects coming from the form. since we are not using it removing